XPlat/Chromium.CroNet/build.cake

```diff
@@ -65,20 +65,24 @@ Task ("externals")
 			}
 
 			EnsureDirectoryExists("./externals/ios");
-			if (CocoaPodVersion () < new System.Version (1, 0))
+
+			if ( ! IsRunningOnWindows() )
 			{
-				CocoaPods.RemoveAt (1);
-			}
+				if (CocoaPodVersion () < new System.Version (1, 0))
+				{
+					CocoaPods.RemoveAt (1);
+				}
 
-			FileWriteLines ("./externals/ios/Podfile", CocoaPods.ToArray ());
-			
-			CocoaPodRepoUpdate ();
+				FileWriteLines ("./externals/ios/Podfile", CocoaPods.ToArray ());
+				
+				CocoaPodRepoUpdate ();
```

Comment:
Don't know if it matters for you, but indentation is different here?

Reply:
Could be. I think I opened it in VS Code and I was working in VS2019. Not sure